declarative/syncprofilewatcher.h

```diff
@@ -42,10 +42,31 @@ class Q_DECL_EXPORT SyncProfileWatcher: public QObject
     Q_PROPERTY(QVariantList log READ log NOTIFY logChanged)
     Q_PROPERTY(SyncSchedule schedule READ schedule NOTIFY scheduleChanged)
     Q_PROPERTY(QVariantMap keys READ keys NOTIFY keysChanged)
-    Q_PROPERTY(Sync::SyncStatus syncStatus READ syncStatus NOTIFY syncStatusChanged)
+    Q_PROPERTY(Status syncStatus READ syncStatus NOTIFY syncStatusChanged)
     Q_PROPERTY(bool synchronizing READ synchronizing NOTIFY syncStatusChanged)
 
 public:
+    enum Status
+    {
+     SYNC_QUEUED = Sync::SYNC_QUEUED,
```

Comment:
Please use indentation of 4. Starting brace to same line as enum.

For consideration if qml api should have CamelCased naming. Makes a bit more delta but could be more conventional there.

Reply:
You're right. I've corrected the indentation and moved to CamelCase.

I've removed the Sync prefix in the enum values. Indeed, there will be used from a class that is called SyncProfileWatcher, and will often be used in conjunction with the syncStatus property. Like that tests like `syncStatus == SyncProfileWatcher.Error` will be less repetitive.